modyn/models/gpt2/others.py

```diff
@@ -0,0 +1,128 @@
+import random
+from typing import Any
+
+import torch
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    RobertaForMaskedLM,
+    T5ForConditionalGeneration,
+)
+
+from modyn.models.coreset_methods_support import CoresetSupportingModule
+
+
+class Roberta:
```

Comment:
why does all of this live inside gpt2/others? i think the PR might need some more cleaning/splititng